src/main/java/seedu/address/model/person/Person.java

```diff
@@ -57,6 +57,10 @@ public Address getAddress() {
         return address;
     }
 
+    public Analytics getAnalytics() {
+        return Analytics.getAnalytics(loanRecords);
```

Comment:
It seems that each time the analytics are viewed, it is reconstructed. Is it possible to refactor this in the future such that it is maintained and modified?

Reply:
It's possible, but it's a trade-off between memory and runtime.